### DISCUSSION TITLE
[977950]冰与火之舞 仅本体

### DISCUSSION THREAD

**Discussion by cyjsnd** [pjy612/SteamManifestCache#646] [Share]
### Steam 游戏ID

977950

### 密钥列表

```hash
977953;cff06fc131de80d2131b228a0ec443303aa7b4fa8bf9b71b02ce9faa59f005c2
977950;900c284c78f11289033d308ddc33fcb65659b670bd2f3684b07c15180e02e971
```

### 清单附件

[【977950】冰与火之舞.zip](https://github.com/pjy612/SteamManifestCache/files/15219415/97750.zip)

### 备注

C2597DA367ECDD91
3387295585@qq.com

**Comment by pjy612** [comment]
看成 冰与火之歌，然后 脑子里联想的是 森林冰火人...

**Reply by pjy612** [reply]
* 标题 Appid 写错了
* appid 写错了
* 清单文件名写错了（不重要）
* 总结只有秘钥对了。。。

**Reply by cyjsnd** [reply]
以改
